integreat_cms/cms/models/push_notifications/push_notification.py

```diff
@@ -50,6 +51,17 @@ class PushNotification(models.Model):
         ),
     )
 
+    @property
```

Comment:
```suggestion
    @cached_property
```
Can you cache this property?